ci/test/pipeline.template.yml

```diff
@@ -638,6 +638,50 @@ steps:
           args: [--scenario=KafkaUpsertUnique]
     coverage: skip
 
+  - id: sqlancer-pqs
+    label: "SQLancer PQS"
```

Comment:
PQS means ...?

Reply:
Pivot Query Synthesis: https://arxiv.org/abs/2001.04174 You don't really have to know the meaning as a dev except that "this is a different approach for finding wrong-results-bugs".